packages/core/src/models/uischema.ts

```diff
@@ -213,6 +213,11 @@ export interface ControlElement extends UISchemaElement, Scopable {
    * An optional label that will be associated with the control
    */
   label?: string | boolean | LabelDescription;
+  /**
+   * The i18n key for the control. It is used to identify the string that needs to be translated.
+   * It is suffixed with `.label`, `.description` and `.error.<keyword>` to derive the corresponding message keys for the control.
```

Comment:
There can be many more suffixes depending on each control. I would only give them as examples.